### DISCUSSION TITLE
Question about versioning the helmfile config.

### DISCUSSION THREAD

**Discussion by bgiegel** [helmfile/helmfile#1356] [Questions]
Hello again,

I have another question. How do you manage the versioning of your helmfile configurations typically ? 

Because in the Chart.yaml we have :

- version: Specify the version of the helm chart
- appVersion: Specify the version of the application itself

Typically in our case the version of the app and the chart are the same because they are hosted in the same repository. So in my base helm charts (the ones I’ve put in the `repositories` field of helmfile) I put the same version for both appVersion and Version.

But still when the chart is deployed we don’t know which version of the values are set in it. And these values are specific to each release. So how do you manage the version of your helmfile config usually ?

In our case all the helmfile configs are set in github repo. So I was thinking to just just use the version of that github repo. Put it into  an environment variable and then use that env variable to set some labels into the kubernetes manifests. What do you think ?

**Comment by yxxhero** [comment]
@bgiegel try to do as you said. please leave comments if you have more issues.
